### DISCUSSION TITLE
How to get started with Open source with EddieHub

### DISCUSSION THREAD

**Discussion by illusiveCode** [EddieHubCommunity/support#1385] [General]
How to get started with Open source with EddieHub

**Comment by kaiwalyakoparkar** [comment]
Hey :wave: Welcome to the open source community

**Comment by kaiwalyakoparkar** [comment]
Key points to take a look at:
- See Eddie's channel if you want to know about the community, techs, and gain some knowledge.
- See the repos in the EddieHub community which matches your interest
- If you are first-timer its a good start by contributing your name to hacktoberfest practice repo.
- Get in conversation with a community member on topics like what project you are starting or simply what project are you interested to contribute to.

**Reply by illusiveCode** [reply]
For sure! Thanks for sharing the tips.

**Comment by eddiejaoude** [comment]
Good place to start are:

1. Practice the contribution and pull request workflow with the [hackatoberfest](https://github.com/EddieHubCommunity/hacktoberfest-practice) repo

**Reply by illusiveCode** [reply]
Thank you, I will do it!

**Reply by eddiejaoude** [reply]
Also this is my GitHub video with lots of tips, have you seen it yet? https://www.youtube.com/watch?v=SYtPC9tHYyQ

**Reply by illusiveCode** [reply]
Will watch today thank you for your support Eddie.
